wazo_call_logd/plugins/cdr/api.yml

```diff
@@ -338,8 +355,8 @@ parameters:
   recording_uuid:
     name: recording_uuid
     description: UUID of the recording
-    required: true
-    type: integer
+    # required: true
```

Comment:
Is this commented line an error?

Reply:
Definitely, I forgot to push the fixing commit. Fixed now.